docs/iot/iot-partners/index.md

```diff
@@ -1,27 +1,26 @@
 # Tizen IoT with Nubison Cloud
 
-The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an intrinsic capability of seamless connectivity.
+The Nubison IoT cloud is a cloud-based IoT platform that provides smart solutions and intelligent services by connecting various things easily, as well as collecting and analyzing IoT data in real time.
+
 Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.
 
-**Figure: Architecture**
+The following diagram shows the interface between Tizen IoT device and Nubison cloud through Nubison plugin:
 
-![Architecture](media/architecture.png)
 
-Compatibility
-- Platform : Tizen 5.0 IoT Headed & Headless and later
-- SDK : Tizen Studio 3.0 and later 
+**Figure: Architecture**
 
+![Architecture](media/architecture.png)
 
-## What is Nubison IoT?
-Nubison IoT provides various IoT features and services which can be utilized in different IoT devices. Data analysis service is one of distinguished IoT services that Nubison provides.
+The Nubison plugin can be installed on Tizen IoT devices compatible with as follow:
 
+* Platform : Tizen 5.0 IoT Headed & Headless and later
+* SDK : Tizen Studio 3.0 and later
 
-References
+For more information about Nubison IoT cloud service, see the following references:
```

Comment:
insert a space between 19 and 20

Reply:
done :)

Reply:
You don't need to an empty line.
The HTML converter converts an empty line before a new list is started,

Reply:
Thanks @journey2w  :to be careful with formatting, I asked to add a line. I will remember that our Html conversion process takes care of lists by itself ! 

Thanks again